src/main/java/org/mafagafogigante/dungeon/schema/JsonRule.java

```diff
@@ -0,0 +1,14 @@
+package org.mafagafogigante.dungeon.schema;
+
+import com.eclipsesource.json.JsonValue;
+
+/**
+ * General interface for json value validation rules.
+ */
+public interface JsonRule {
+
+  /**
+   * Key method for json validation.
+   */
+  JsonValidationResult validate(JsonValue validationElement);
```

Comment:
I don't even know if this is in the Dungeon Java Style or not, but I like having a blank line before closing a class (this is **really** minor).